vslib/SwitchStateBase.h

```diff
@@ -108,6 +108,8 @@ namespace saivs
                     _In_ uint32_t sys_port_count,
                     _In_ const sai_system_port_config_t *sys_port_cfg_list);
 
+            sai_status_t create_voqs();
+            sai_status_t create_voq_per_sysport(_In_ sai_object_id_t sys_port_id);
```

Comment:
move paremater to second line like in other functions

Reply:
I fixed the parameter. The coverage seems to be not detecting the coverage added my new test. I checked in my local environment, the new functions are exercised by the test I added. @kcudnik